content/c-sharp/concepts/math-functions/terms/max/max.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Math.Max()'
+Description: 'Returns the larger of two specified numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Max()`** method returns the larger of two specified numbers.
+
+## Syntax
+
+```pseudo
+//returns the greater value from the two numbers
+Math.Max(num1, num2)
+```
+
+## Example
+
+The following example compares two integers `i1` and `i2` and writes greater to the console.
+
+```cs
+using System;
+
+class Program {
+  public static void Main (string[] args) {
+    int i1 = -234, i2 = 345;
+
+    Console.WriteLine("Math.Max (int) = " + Math.Max(i1,i2));
+  }
+}
+```
+
```

Comment:
Following the example, we'd like a ```shell block showing the output, introduced by a sentence like "This example results in the following output:"

Reply:
I have also added  a ```shell example output to codebyte example at the bottom of the file. Not sure if it is needed.

Reply:
So i copied the changed file contents from my local file here to the github webpage and should be ready for another review. Thanks in advance.